web/src/beta/lib/core/StoryPanel/SelectableArea/hooks.ts

```diff
@@ -43,12 +27,9 @@ export default ({ editMode, isSelected, property, setEditMode, onClickAway }: Pr
   }, [property?.panel]);
 
   return {
-    isHovered,
     showPadding,
     panelSettings,
```

Comment:
Can you delete using sending panelSettings here since its not being used anymore 🙏🏻 After that you can close this comment 👍🏻